ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -476,8 +482,16 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_an
     if (conv_config.reshard_if_not_optimal || needs_shard_or_reshard) {
         auto block_shard_orientation =
             conv_config.transpose_shards ? ShardOrientation::COL_MAJOR : ShardOrientation::ROW_MAJOR;
-        ParallelConfig optimal_parallel_config = determine_parallel_config(
-            shard_layout, batch_size, in_channels, height, width, out_channels, device, block_shard_orientation);
+        const ParallelConfig& optimal_parallel_config = determine_parallel_config(
```

Comment:
seems like a bad rebase/merge?

Reply:
most probably auto formated during commit of this file

Reply:
`const ParallelConfig& ` we don't need at const & part here